lib/client.js

```diff
@@ -99,23 +100,34 @@ extend(Raven.prototype, {
       globalContext.extra = options.extra;
     }
 
+    this.onFatalError = this.defaultOnFatalError = function (err, sendErr, eventId) {
+      console.error(err.stack);
```

Comment:
good catch, I was just doing `console.error(err)` but changed to `err.stack` for some goofy domain cases, will make this more defensive